docs/source/databases.rst

```diff
@@ -18,7 +18,7 @@ NCBI
 Markers
 =======
 
-- bacteria single-copy-markers - `link <https://github.com/WiscEvan/Autometa/raw/dev/databases/markers/bacteria.single_copy.hmm>`__
-- bacteria single-copy-markers cutoffs - `link <https://raw.githubusercontent.com/WiscEvan/Autometa/dev/databases/markers/bacteria.single_copy.cutoffs?token=AGF3KQVL3J4STDT4TJQVDBS6GG5FE>`__
-- archaea single-copy-markers - `link <https://github.com/WiscEvan/Autometa/raw/dev/databases/markers/archaea.single_copy.hmm>`__
-- archaea single-copy-markers cutoffs - `link <https://raw.githubusercontent.com/WiscEvan/Autometa/dev/databases/markers/archaea.single_copy.cutoffs?token=AGF3KQXVUDFIH6ECVTYMZQS6GG5KO>`__
+- bacteria single-copy-markers - `link <https://raw.githubusercontent.com/KwanLab/Autometa/dev/autometa/databases/markers/bacteria.single_copy.hmm>`__
```

Comment:
This may be a larger issue that needs to be addressed in the docs (cc @Sidduppal )
Once dev is merged into main, dev docs will contain `main` anyways

Reply:
If there's not an issue for that already it might be worth creating one as a reminder

Reply:
Yeah, that works. There will be a number of merge conflicts to address during that merge, so I was thinking that would be a sufficient reminder to double check `dev` being changed to `main` in all files. Either works 🤷